managers/s3.py

```diff
@@ -83,81 +65,64 @@ def generate_manifest(self, record, source_url, manifest_url):
+            if object_extension == 'epub':
+                self.store_epub(object, key, bucket)
 
-            objectType = mimetypes.guess_type(objKey)[0]\
-                or 'binary/octet-stream'
+            object_type = mimetypes.guess_type(key)[0] or 'binary/octet-stream'
 
-            return self.s3Client.put_object(
-                ACL=bucketPermissions,
-                Body=obj,
+            return self.client.put_object(
+                ACL=bucket_permissions,
+                Body=object,
                 Bucket=bucket,
-                Key=objKey,
-                ContentMD5=objMD5,
-                ContentType=objectType,
-                Metadata={'md5Checksum': objMD5}
+                Key=key,
+                ContentMD5=object_md5,
+                ContentType=object_type,
+                Metadata={ 'md5Checksum': object_md5 }
             )
         except ClientError as e:
-            raise S3Error(f'Unable to store file {objKey} in s3: {e}')
+            raise S3Error(f'Unable to store file {key} in s3: {e}')
 
-    def putExplodedEpubComponentsInBucket(self, obj, objKey, bucket):
-        keyRoot = '.'.join(objKey.split('.')[:-1])
+    def store_epub(self, object, key: str, bucket: str):
+        key_prefix = '.'.join(key.split('.')[:-1])
 
-        with ZipFile(BytesIO(obj), 'r') as epubZip:
-            for component in epubZip.namelist():
-                componentObj = epubZip.open(component).read()
-                componentKey = '{}/{}'.format(keyRoot, component)
-                self.putObjectInBucket(componentObj, componentKey, bucket)
+        with ZipFile(BytesIO(object), 'r') as epub_zip:
+            for component in epub_zip.namelist():
+                self.put_object(object=epub_zip.open(component).read(), key=f'{key_prefix}/{component}', bucket=bucket)
 
-    def getObjectFromBucket(self, objKey, bucket, md5Hash=None):
+    def get_object(self, key: str, bucket: str, md5_hash=None):
         try:
-            if md5Hash:
-                return self.s3Client.get_object(
+            if md5_hash:
+                return self.client.get_object(
                     Bucket=bucket,
-                    Key=objKey,
-                    IfNoneMatch=md5Hash
+                    Key=key,
+                    IfNoneMatch=md5_hash
                 )
             
-            return self.s3Client.get_object(Bucket=bucket, Key=objKey)
+            return self.client.get_object(Bucket=bucket, Key=key)
         except ClientError:
             raise S3Error('Unable to get object from s3')
-        
-    def load_batches(self, objKey, bucket):
-
-        '''# Loading batches of data using a paginator until there are no more batches'''
-
-        paginator = self.s3Client.get_paginator('list_objects_v2')
-        page_iterator = paginator.paginate(Bucket=bucket, Prefix=objKey)
-        return page_iterator
 
     @staticmethod
-    def getmd5HashOfObject(obj):
+    def get_md5_hash(object):
```

Comment:
just a note: `object` is a reserved work in python (everything inherits from it) - *probably* won't cause any issues, but might trigger some linters / get highlighted weirdly